share/spice/currency/currencyNames.txt

```diff
@@ -156,7 +156,7 @@ vuv,ni-vanuatu vatu,vanuatu vatu,
 wst,samoan tala,samoa tala,
 xaf,central african cfa franc beac,re africaine cfa franc beac,
 xag,silver ounce,silver,
-xau,gold ounce,gold,
+xau,gold ounce,gold,1 oz gold in usd,gold price,gold prices today,gold price per ounce,1oz gold,1oz gold in usd,1 ounce gold in usd, 1 ounces gold in usd
 xbt,bitcoin,btc,bitcoins
```

Comment:
@gabhi Thanks for this contribution. I'm not sure this is the right way to proceed as this file should only need to contain the currency/metal name and the additional words `in` `today` should be handled by the Currency.pm code itself https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Currency.pm#L25-L33 @moollaza ?